markdown/templates.go

```diff
@@ -80,11 +80,32 @@ func execute(fileName string, data any, outputFilePath string) error {
 // all templates
 func getTemplateHelpers() template.FuncMap {
 	return template.FuncMap{
-		"lower":  strings.ToLower,
-		"upper":  strings.ToUpper,
-		"pascal": strcase.ToCamel,
-		"camel":  strcase.ToLowerCamel,
-		"snake":  strcase.ToSnake,
-		"kebab":  ToKebabCase,
+		"lower":     strings.ToLower,
+		"upper":     strings.ToUpper,
+		"pascal":    strcase.ToCamel,
+		"camel":     strcase.ToLowerCamel,
+		"snake":     strcase.ToSnake,
+		"kebab":     ToKebabCase,
+		"multiArgs": multiArgs,
 	}
 }
+
+// multiArgs is a helper function that can be used in templates to create a
+// map of key value pairs. It takes an even number of arguments and returns a
+// map of the arguments. The first argument is the key and the second is the
+// value. The third argument is the next key and so on. This is useful for
+// passing multiple arguments to a nested template.
+func multiArgs(values ...interface{}) (map[string]interface{}, error) {
```

Comment:
Nice, I really like Go templates and how you can easily inject helper functions like that. Makes the templates very readable.